app/scripts/lib/app-start.js

```diff
@@ -98,36 +100,16 @@ define(function (require, exports, module) {
         this.initializeL10n(),
         this.initializeInterTabChannel()
       ])
-      .then(_.bind(this.allResourcesReady, this))
-      .fail(function (err) {
-        if (console && console.error) {
-          console.error('Critical error:');
-          console.error(String(err));
-        }
-
-        // if there is no error metrics set that means there was probably an error with app start
-        // therefore force error reporting to get error information
-        if (! self._sentryMetrics) {
-          self.enableSentryMetrics();
-        }
-
-        self._sentryMetrics.captureException(err);
```

Comment:
@shane-tomlinson Are you sure we want to move this?

Here's what happens with the error report if you move this:

### Before

![image](https://cloud.githubusercontent.com/assets/128755/12583497/1aad8640-c411-11e5-9d02-30706632199b.png)


### After

![image](https://cloud.githubusercontent.com/assets/128755/12583482/03f6c6c8-c411-11e5-97d5-3571181de227.png)

Seems like we lose the name of the app start init function that failed this way.